shared-libs/rules-engine/test/rules-state-store.spec.js

```diff
@@ -34,7 +35,7 @@ describe('rules-state-store', () => {
 
     const isDirty = rulesStateStore.isDirty('a');
     expect(isDirty).to.be.true;
-    expect(rulesStateStore.currentUser()).to.eq(userDoc);
+    expect(rulesStateStore.currentUserContact()).to.eq(userDoc);
```

Comment:
can we inject a setting stub and assert on the value of `currentUserSetting()` here also?  not super important, but nice to round out the tests and make assertions symetrical

Reply:
Done.